core/server/adapters/storage/utils.js

```diff
@@ -1,27 +1,47 @@
-var globalUtils = require('../../utils');
+var debug = require('ghost-ignition').debug('storage:utils'),
+    globalUtils = require('../../utils');
 
 /**
  * @TODO: move `index.js` to here - e.g. storageUtils.getStorage
  */
 
 /**
  * Sanitizes a given URL or path for an image to be readable by the local file storage
+ * as storage needs the path without `/content/images/` prefix
  * Always returns {string} url
  * @param {string} imagePath
  * @returns {string} imagePath
  * @description Takes a url or filepath and returns a filepath with is readable
  * for the local file storage.
  */
 exports.getLocalFileStoragePath = function getLocalFileStoragePath(imagePath) {
+    debug('requested imagePath:', imagePath);
+
     if (imagePath.match(new RegExp('^' + globalUtils.url.urlJoin(globalUtils.url.urlFor('home', true), globalUtils.url.getSubdir(), '/', globalUtils.url.STATIC_IMAGE_URL_PREFIX)))) {
-        // Storage needs the path without `/content/images/` prefix
         // The '/' in urlJoin is necessary to add the '/' to `content/images`, if no subdirectory is setup
+
         return imagePath.replace(new RegExp('^' + globalUtils.url.urlJoin(globalUtils.url.urlFor('home', true), globalUtils.url.getSubdir(), '/', globalUtils.url.STATIC_IMAGE_URL_PREFIX)), '');
     } else if (imagePath.match(new RegExp('^' + globalUtils.url.urlJoin(globalUtils.url.getSubdir(), '/', globalUtils.url.STATIC_IMAGE_URL_PREFIX)))) {
-        // Storage needs the path without `/content/images/` prefix
         // The '/' in urlJoin is necessary to add the '/' to `content/images`, if no subdirectory is setup
+
         return imagePath.replace(new RegExp('^' + globalUtils.url.urlJoin(globalUtils.url.getSubdir(), '/', globalUtils.url.STATIC_IMAGE_URL_PREFIX)), '');
     } else {
         return imagePath;
     }
 };
+
+/**
+ * @description compares the imagePath with a regex that reflects our local file storage
+ * @param {String} imagePath as URL or filepath
+ * @returns {Boolean}
+ */
+
+exports.isLocalImage = function isLocalImage(imagePath) {
```

Comment:
This function is basically doing what `getLocalFileStoragePath` is doing, except of the return value 🤔 
As these conditions are quite long and it takes a bit to read them, how about doing

```
x = getLocalFileStoragePath(path)
if (x !== path) {
   return true;
}

return false;
```

Reply:
Thank you for this! Totally makes sense to me and looks sooo much better now!